flink-ml-parent/flink-ml-lib/src/main/java/org/apache/flink/ml/common/linalg/BLAS.java

```diff
@@ -19,26 +19,59 @@
 
 package org.apache.flink.ml.common.linalg;
 
+import org.apache.flink.util.Preconditions;
+
 /**
  * A utility class that provides BLAS routines over matrices and vectors.
  */
 public class BLAS {
+
+	// For level-1 routines, we use Java implementation.
 	private static final com.github.fommil.netlib.BLAS NATIVE_BLAS = com.github.fommil.netlib.BLAS.getInstance();
+
+	// For level-2 and level-3 routines, we use the native BLAS.
```

Comment:
nit: use `/* */`

Reply:
Yes, please refine it when merging. Thanks.